ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -609,7 +601,7 @@ bool UncoupledProcessesTimeLoop::loop()
         t = ts.current();
         timestep = ts.steps();
 
-        INFO("=== timestep #%u (t=%gs, dt=%gs) ==============================",
```

Comment:
Though I like the proposed change, I'd prefer to see it in a separate PR so that everybody who is relying on the formatting can update their parsing scripts.